pennylane_lightning/lightning_gpu/lightning_gpu.py

```diff
@@ -46,14 +66,17 @@
             StateVectorMPIC64,
             StateVectorMPIC128,
         )
+        from pennylane_lightning.lightning_gpu_ops.algorithmsMPI import (
```

Comment:
I assume the coverage warning here an below is due to the MPI tests not running? Or do we need to update the tests to potentially hit these at some point?

Reply:
No, we're just not running MPI tests yet.